crates/solidity/src/project/contract/metadata.rs

```diff
@@ -24,14 +26,13 @@ impl Metadata {
         solc_metadata: serde_json::Value,
         solc_version: semver::Version,
         revive_pallet_version: Option<semver::Version>,
-        revive_version: semver::Version,
         optimizer_settings: revive_llvm_context::OptimizerSettings,
     ) -> Self {
         Self {
             solc_metadata,
             solc_version,
```

Comment:
maybe `solc_version` should be also in long format?

Reply:
Ok, changed it